docs/docusaurus/docs/core/connect_to_data/configure_credentials/_access_credentials.md

```diff
@@ -12,4 +12,4 @@ Or you could reference a configured variable that contains the full connection s
 connection_string="${POSTGRES_CONNECTION_STRING}"
 ```
 
-When you pass a string that references your stored credentials to a {GxData.product_name} method that requires a connection string as a parameter the referenced variable will be substituted for the corresponding stored value.
+When you pass a string that references your stored credentials to a GX Core method that requires a connection string as a parameter the referenced variable will be substituted for the corresponding stored value.
```

Comment:
Just preference on hard-coding vs using the variable? Seems like updating the variable would have been quicker? (I have no preference)

Reply:
It would have been, but it didn't render well in Google's search snippets. :/

Reply:
oh weird 🤷